packages/core/src/layout/InfoCard/InfoCard.stories.tsx

```diff
@@ -34,23 +45,23 @@ const Wrapper: FC<{}> = ({ children }) => (
 export const Default = () => (
   <Wrapper>
     <InfoCard title="Information Card">
-      <div style={cardContentStyle} />
+      <div style={cardContentStyle}>{text}</div>
```

Comment:
does this mean that we could remove the kinda ugly card content style entirely?

Reply:
Fixed!